modules/packages/npm/creator.go

```diff
@@ -176,6 +176,10 @@ func ParsePackage(r io.Reader) (*Package, error) {
 			meta.Homepage = ""
 		}
 
+		if meta.Repository.Type != "git" {
+			meta.Repository.URL = ""
+		}
```

Comment:
This may be debatable if we want to display this information in the ui.

Reply:
Yes this is true; I say we remove these lines & display it in the ui, unless somone has something against it? Or should we rather add an config option for this?

Reply:
Removed the resetting of the URL when type isn't git.